README.md

```diff
@@ -133,7 +133,6 @@ mikelangelo/capstan-packages   latest   bee017f1e55c   16 hours ago   2.82 GB
 Capstan-packages platform is distributed under the 3-clause BSD license.
 
 ## Acknowledgements
-This project  has been conducted within the RIA [MIKELANGELO
-project](https://www.mikelangelo-project.eu) (no.  645402), started in January
-2015, and co-funded by the European Commission under the H2020-ICT- 07-2014:
+This project has been conducted within the [MIKELANGELO project](https://www.mikelangelo-project.eu),
```

Comment:
Please add the project number back. This is unique identifier of the projects, co-funded by EC. Also, the wording could be improved. I suggest we start saying "This code has been developed within the ". It is fine to leave it as "Project", but it does look a bit strange.

Reply:
Done. Also, I've applied your comment about the rewording, it really sounds better now.